paasta_tools/kubernetes_tools.py

```diff
@@ -1308,6 +1339,35 @@ def get_volume_mounts(
                 for volume in secret_volumes
             ]
         )
+        if self.config_dict.get("boto_keys", []):
+            secret_hash = self.get_boto_secret_hash()
+            service_name = (
+                self.get_sanitised_service_name()
+                + "-"
+                + self.get_sanitised_instance_name()
+            )
```

Comment:
should this be 
```suggestion
            service_name = self.get_sanitised_deployment_name()
```
for consistency?